app/rankers/algorithms.py

```diff
@@ -127,12 +127,12 @@ def personalize_topic_slates(input_slate_configs: List['SlateConfigModel'],
     """
     topic_to_score_map = {t.curator_topic_label: t.score for t in personalized_topics.curator_topics}
     # filter non-topic slates
-    input_slate_configs = list(filter(lambda s: s.curatorTopicLabel in topic_to_score_map, input_slate_configs))
```

Comment:
I missed that this variable was assigned to, so the input wasn't affected after all. Thanks for the rename!